app/Resources/translations/messages.en.xliff

```diff
@@ -246,7 +246,9 @@ This profile page gives you insight in which personal data, provided by your ins
       </trans-unit>
       <trans-unit id="c4b69512880a8532d9740e870ed8a60b8791b3bc" resname="profile.my_connections.orcid.description">
         <source>profile.my_connections.orcid.description</source>
-        <target>ORCID is a nonproprietary alphanumeric code to uniquely identify scientific and other academic authors. This addresses the problem that a particular author's contributions to the scientific literature or publications in the humanities can be hard to recognize as most personal names are not unique, they can change (such as with marriage), have cultural differences in name order, contain inconsistent use of first-name abbreviations and employ different writing systems.</target>
+        <target>Connect your existing or newly created ORCID iD to SURFconext once. The ORCID iD is a code that is used to uniquely identify scientific and other academic authors.
```

Comment:
Is it intended to be `iD` in this text (small i big D)?

Reply:
Well spotted, copypasta from a text provided by a PM. But will update this!

Reply:
🎉